spec/supa/commands/attribute_spec.rb

```diff
@@ -1,12 +1,12 @@
 require 'spec_helper'
 
 describe Supa::Commands::Attribute do
-  let(:object) { double('DummyArticle', name: 'Zaphod') }
+  let(:object) { double('DummyArticle', name: 'Zaphod', surname: 'Surname', name_as_sym: :zaphod) }
   let(:representer) { Supa::ArticleRepresenter.new(object) }
-  let(:render_result) { attribute.represent }
+  subject(:render_result) { attribute.represent }
```

Comment:
Please read this doc https://github.com/distribusion/standards/wiki/RSpec#subject